src/blockdata/script.rs

```diff
@@ -919,6 +919,11 @@ impl Builder {
         }
     }
 
+    /// Adds instructions to push an XOnly public key onto the stack.
+    pub fn push_x_only_key(self, key: &XOnlyPublicKey) -> Builder {
```

Comment:
note to reviewers, the already existing function is called `push_key` (not `push_public_key`). Therefore, it makes sense to call this `push_x_only_key`.